array-manipulation-1/src/reverse.js

```diff
@@ -0,0 +1,9 @@
+/* exported reverse */
+function reverse(array) {
+  var newArray = [];
+  var realLength = array.length;
+  for (var i = 0; i < realLength; i++) {
+    newArray[i] = array.pop();
+  }
```

Comment:
`.pop()` actually wrecks the original array, which we want to avoid. Instead, try looping backwards and `.push()`ing into the new array.